File: java-main/java-main/Array/Ex3/p2.java
//WAP to reverse each element in an array.
import java.io.*;

class Demo{

	public static void main(String[] args)throws IOException{

		BufferedReader br= new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter array size");

		int size=Integer.parseInt(br.readLine());

		int arr[]=new int[size];

		System.out.println("Enter array elements");

		for(int i=0;i<size;i++){
			arr[i]=Integer.parseInt(br.readLine());
		}

		System.out.println("Reverse numbers are: ");

		for(int i=0;i<size;i++){
			revNum(arr[i]);
		}
	}

	static void revNum(int x){

		int rev=0;

		while(x!=0){

			int rem=x%10;
			rev=rev*10+rem;
			x=x/10;
		}
		System.out.println(rev + " ");
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Sorting/p2.c
#include<stdio.h>
void bubblesort(int arr[],int size){
	for(int i=0; i<size-1; i++){
		
		for(int j=0; j<size-i-1; j++){
			
			if(arr[j]>arr[j+1]){
				
				int temp=arr[j];
				arr[j]=arr[j+1];
				arr[j+1]=temp;
			}
		}	
	}
}
void main(){
	
	int size;
	printf("Enter Size of an Array\n");
	scanf("%d",&size);

	int arr[size];
	printf("Enter Element\n");
	for(int i=0; i<size; i++){
		scanf("%d",&arr[i]);
	}

	bubblesort(arr,size);
	
	printf("Sorted Array Are\n");
	for(int i=0; i<size; i++){
		printf("%d\t\n",arr[i]);
	}
}
